docs/book/src/reference/metrics.md

```diff
@@ -69,6 +69,37 @@ NAME                                         AGE
 monitor-controller-manager-metrics-monitor   2m8s
 ```
 
+### RBAC for Prometheus
+
+If you are using
+[kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) or
+[prometheus-operator](https://github.com/prometheus-operator/prometheus-operator)
+for cluster monitoring and you have trouble seeing your manager metrics in
+Prometheus, check the Prometheus Targets page for the resource 
+`serviceMonitor/<namespace>/<project-prefix>-controller-manager-metrics-monitor`.
+
+If you can't find it, you need to give extra permissions to the Prometheus
+ServiceAccount.
+
+> **⚠ WARNING**  
+> A ClusterRole for the Prometheus `prometheus-k8s` ServiceAccount in the
```

Comment:
By default, A ClusterRole will be created for the Prometheus `prometheus-k8s` ServiceAccount in the
`monitoring` namespace. Please ensure that the ServiceAccount exists and that it belongs to Prometheus. If the name or namespace ?of Prometheus? are different, you should change them in
`config/prometheus_rbac/cluster_role_binding.yaml` before applying.

"Please ensure that the ServiceAccount exists and that it belongs to Prometheus" - what does it mean? How do they make sure of that?